src/app/pages/PerpetualPage/hooks/graphql/useGetTraderEvents.ts

```diff
@@ -194,6 +195,7 @@ class EventDictionary {
           'rate8h',
           'blockTimestamp',
           'deltaTime',
+          'fundingPayment{ position{ perpetual{ id } } }', //TODO: modify the graph to extract the perpId more straight forward
```

Comment:
This is not a TODO for us but for Remy/Betsy so you need to discuss it with them if you want (I personally think that the current way how we store it is good).